### PR TITLE
Added HostPath, ImagePullSecret and TerminationGracePeriodSeconds support

### PR DESCRIPTION
**Pull request by ofirmakmal**
Added additional fields needed for the Helm chart.

### CODE REVIEW

**Comment by vincentsarago** [comment]
@ividito @ranchodeluxe would you be able to review this PR? 🙏

**Review by geospatial-jeff — APPROVED** [review]
LGTM

File: deployment/k8s/charts/values.yaml [hunk 1]
```diff
@@ -26,6 +28,15 @@ ingress:
...
 
+extraHostPathMounts: []
+  # - name: map-sources
+  #   mountPath: /map-sources/
+  #   hostPath: /home/ubuntu/map-sources
+  #   readOnly: false
+  #   mountPropagation: HostToContainer # OPTIONAL
+
+imagePullSecrets: []
+
```
Comment (ranchodeluxe) [inline deployment/k8s/charts/values.yaml:39]: nitpick: but I like what you did above with `extraHostPathMounts` showing an example of what should go there. Can we add `# -name: regcred` here also please as an example?

**Review by ranchodeluxe — APPROVED** [review]
LGTM. Left a tiny comment that can be ignored or updated

**Pull request merged**